PerformanceMetrics/PerfMetricsWebKitBrowser.config

```diff
@@ -1,7 +1,11 @@
 set(autostart ${PLUGIN_PERFORMANCEMETRICS_AUTOSTART})
 
 map()
-    kv(callsign "WebKitBrowser")
+    if( NOT "${PLUGIN_PERFORMANCEMETRICS_RDKBROWSER}" STREQUAL "")
```

Comment:
As mentioned above I would make it boolean flag as it can only have two values and the just hardcode the key/value pair

Reply:
I set the key based on the boolean flag